CMakeLists.txt

```diff
@@ -42,10 +42,6 @@ endif()
 
 project(memgraph LANGUAGES C CXX)
 
-# NOTE: once in a while this needs to be toggled to check headers are
-#       correct and PCH isn't masking any include issues
-set(CMAKE_DISABLE_PRECOMPILE_HEADERS OFF)
```

Comment:
Is it intended change?

Reply:
Yes, we introduce PCH + unity builds, its caused multiple issues. Removing it for now. Putting it in this PR to reduce number of PRs